cardano-node/src/Cardano/Node/TUI/LiveView.hs

```diff
@@ -492,7 +492,7 @@ initLiveViewState = do
 
     return $ LiveViewState
                 { lvsScreen                 = MainView
-                , lvsRelease                = "Byron"
+                , lvsRelease                = "Shelley"
```

Comment:
This change would also affect the node running on the mainnet which is still `Byron`.

Either this should stay as is until we actually do switch mainnet to Shelley of it should get the correct value for that from somewhere further up the software stack.